sdk/tables/azure-data-tables/inc/azure/data/tables/serializers.hpp

```diff
@@ -2,38 +2,18 @@
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
 #pragma once
-
-#include <azure/core/context.hpp>
-#include <azure/core/credentials/credentials.hpp>
-#include <azure/core/http/http.hpp>
-#include <azure/core/http/policies/policy.hpp>
-#include <azure/core/internal/http/pipeline.hpp>
 #include <azure/core/internal/json/json.hpp>
-#include <azure/core/response.hpp>
-#include <azure/core/url.hpp>
-#include <azure/data/tables/dll_import_export.hpp>
+#include <azure/core/xml.hpp>
 #include <azure/data/tables/models.hpp>
-#include <azure/data/tables/rest_client.hpp>
-#include <azure/data/tables/rtti.hpp>
-#include <azure/data/tables/transactions.hpp>
-#include <azure/storage/common/crypt.hpp>
-#include <azure/storage/common/internal/constants.hpp>
-#include <azure/storage/common/internal/shared_key_policy_lite.hpp>
-#include <azure/storage/common/internal/storage_bearer_token_auth.hpp>
-#include <azure/storage/common/internal/storage_per_retry_policy.hpp>
-#include <azure/storage/common/internal/storage_service_version_policy.hpp>
-#include <azure/storage/common/internal/storage_switch_to_secondary_policy.hpp>
-#include <azure/storage/common/internal/xml_wrapper.hpp>
-#include <azure/storage/common/storage_common.hpp>
-#include <azure/storage/common/storage_credential.hpp>
 
 #include <cstdint>
 #include <memory>
 #include <string>
 #include <utility>
 #include <vector>
 
-namespace Azure { namespace Data { namespace Tables {
+namespace Azure { namespace Data { namespace Tables { namespace _detail {
```

Comment:
This file does only have definitions in `_detail` namespace now. Therefore, it should be moved to `internal` directory (yes, I know, it is `detail`, but still, as of directories, both headers containing only `_detail` and `_internal` namespaces, do live in `inc/.../internal`).